### ISSUE TITLE
Committer name

### ISSUE THREAD

**Issue by Adarma** [kemayo/sublime-text-git#480]
Every time i commit I get the following message in the panel:

> [branch abcdef] Commit Message
>  Committer: MyName my.email@my.domain.com
> Your name and email address were configured automatically based
> on your username and hostname. Please check that they are accurate.
> You can suppress this message by setting them explicitly. Run the
> following command and follow the instructions in your editor to edit
> your configuration file:
> 
> ```
> git config --global --edit
> ```
> 
> After doing this, you may fix the identity used for this commit with:
> 
> ```
> git commit --amend --reset-author
> ```
> 
>  2 files changed, 74 insertions(+), 22 deletions(-)

I have my `[user] name` and `[user] email` set in the global config correctly.
I don't get the message if I commit from the command line.

I can see from `git log` the different name and email depending on if I commit via the commandline or the sublime-text-git

Could it be because my home folder is set to H: drive (H:.gitconfig)?

**Comment by marvink87** [comment]
The cause of this is in "git/__init__.py" Currently on line 199 - 205:
```
            env = os.environ.copy()

            shell = False
            if sublime.platform() == 'windows':
                shell = True
                if 'HOME' not in env:
                    env[str('HOME')] = str(env['USERPROFILE'])
```
I'm not sure what this is a workaround for as without "HOME" it works fine. Setting HOME to HOMEDRIVE + HOMEPATH works much better for me.

I've currently worked around this issue by manually setting the "HOME" variable

**Comment by kemayo** [comment]
At least according to [Microsoft's docs](https://technet.microsoft.com/en-us/library/cc749104(v=ws.10).aspx), `USERPROFILE` should be the same thing as `HOMEDRIVE`+`HOMEPATH`...

**Comment by mosbasik** [comment]
My home folder is on the U: drive.  Checking my environment variables with Git Bash shows this:

    $ env | grep HOME
    HOMEPATH=\
    HOME=/u/
    HOMEDRIVE=U:

and:

    $ env | grep USERPROFILE
    USERPROFILE=C:\Users\<username redacted>

so I think the Microsoft docs might be incorrect.  My gitconfig with name and email are in `U:\.gitconfig` (or `/u/.gitconfig` depending on which shell you ask), and just like OP, any commits I make from the terminal are able to use that config just fine.

It looks like @marvink87 has submitted in a PR (#502) that would fix my issue as well - I'd love to see it accepted! (Thanks marvin!)

**Comment by mosbasik** [comment]
As a workaround in the meantime, I ended up symlinking `.gitconfig` and `.ssh/` to where the extension expects them to be.  Maybe this will help someone else looking:

1. Open cmd.exe as an administrator (several ways to do this, one is <kbd>Windows</kbd>-<kbd>X</kbd> and select `Command Prompt (Admin)`

2. Run these commands.  Note that `mklink`'s argument order is `mklink [LINK] [TARGET]`, which the reverse of `ln` argument order if you're more familiar with that.  Also note that my "true" home folder that I thought of as being on U: is actually mapped to a network share - so I needed to put in the actual path, not U:

        mklink C:\Users\<username>\.gitignore \\network.share.domain\folder\folder\.gitignore
        mklink /D C:\Users\<username>\.ssh \\network.share.domain\folder\folder\.ssh
